crates/api_common/src/community.rs

```diff
@@ -20,6 +23,12 @@ pub struct GetCommunityResponse {
   pub community_view: CommunityView,
   pub moderators: Vec<CommunityModeratorView>,
   pub online: usize,
+  /// Metadata of the instance where the community is located. Only fields name, sidebar,
+  /// description, icon, banner, actor_id, last_refreshed_at get federated, everything else uses
+  /// default values. May be null if the community is hosted on an older Lemmy version, or on
+  /// another software.
+  /// TODO: this should probably be SiteView
+  pub site: Option<Site>,
```

Comment:
I wasnt sure whats the best way to expose this data over the api. Anyway its not needed for federated bans, so i can just remove it and implement it properly later. The main purpose is to show the instance sidebar (with rules etc) when browsing a remote community.